url-shortener/src/main/resources/application.yml

```diff
@@ -0,0 +1,10 @@
+spring:
+  datasource:
+    username: ju22user
+    password: ju22pass
+    url: jdbc:postgresql://93.175.204.87:5432/postgres?currentSchema=romantsev
+    driver-class-name: org.postgresql.Driver
+  jpa:
+    hibernate:
+      ddl-auto: update
```

Comment:
@vromantsev you rely on Flyway, to update the schema. So you should not use `ddl-auto: update`.

Reply:
removed ddl-auto: update